src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -4,6 +4,7 @@ import React, { Component } from "react"
 import styled from "styled-components"
 import { ReadMore } from "Styleguide/Components/ReadMore"
 import { Col, Grid, Row } from "Styleguide/Elements/Grid"
+import { slugify } from "underscore.string"
```

Comment:
Not sure how this passed the `type-check` CI phase, locally I’m getting a failure because this package was not added as a dependency to Reaction.

Reply:
Added in https://github.com/artsy/reaction/pull/1597

Reply:
@eessex want to pair on removing this dependency? The slug for a collection should come from the API response, trying to re-create it from the client will end up inconsistent

Reply:
This is an id for a collection category, which doesn't actually have a slug -- I have the dependency in my local reaction right now?

Reply:
I'm open to adding unique ids to the category, but will require some additional work in Kaws cc @l2succes

Reply:
> I have the dependency in my local reaction right now?

Might have been a transitive dependency of another dependency that got updated/removed? Force has a hard dependency on it, so the code does work when used inside Force, but Reaction itself did not have one so not sure how else that could have worked… Unless maybe you ran `npm install underscore.string`?